src/tests/classic/run.rs

```diff
@@ -534,6 +534,123 @@ fn test_treehash_constant_embedded_modern_loop() {
     assert!(result_text.contains("stack limit exceeded"));
 }
 
+#[test]
+fn test_embed_file_2() {
+    let program = do_basic_run(&vec![
+        "run".to_string(),
+        "-i".to_string(),
+        "resources/tests".to_string(),
+        "(mod () (embed-file testhex hex hex-embed-01.hex) testhex)".to_string(),
+    ])
+    .trim()
+    .to_string();
+    let run_result = do_basic_brun(&vec!["brun".to_string(), program, "()".to_string()])
+        .trim()
+        .to_string();
+    assert_eq!(run_result, "(65 66 67)");
+}
+
+#[test]
+fn test_embed_file_4() {
+    let program = do_basic_run(&vec![
+        "run".to_string(),
+        "-i".to_string(),
+        "resources/tests".to_string(),
+        "(mod () (include *standard-cl-21*) (embed-file testhex hex hex-embed-01.hex) testhex)"
+            .to_string(),
+    ])
+    .trim()
+    .to_string();
+    let run_result = do_basic_brun(&vec!["brun".to_string(), program, "()".to_string()])
+        .trim()
+        .to_string();
+    assert_eq!(run_result, "(65 66 67)");
+}
+
+#[test]
+fn test_embed_file_5() {
+    let program = do_basic_run(&vec![
+        "run".to_string(),
+        "-i".to_string(),
+        "resources/tests".to_string(),
+        "(mod () (embed-file testsexp sexp embed.sexp) testsexp)".to_string(),
+    ])
+    .trim()
+    .to_string();
+    let run_result = do_basic_brun(&vec!["brun".to_string(), program, "()".to_string()])
+        .trim()
+        .to_string();
+    assert_eq!(run_result, "(lsh 24 25)");
+}
+
+#[test]
+fn test_embed_file_6() {
+    let program = do_basic_run(&vec![
+        "run".to_string(),
+        "-i".to_string(),
+        "resources/tests".to_string(),
+        "(mod () (include *standard-cl-21*) (embed-file testsexp sexp embed.sexp) testsexp)"
+            .to_string(),
+    ])
+    .trim()
+    .to_string();
+    let run_result = do_basic_brun(&vec!["brun".to_string(), program, "()".to_string()])
+        .trim()
+        .to_string();
+    assert_eq!(run_result, "(lsh 24 25)");
+}
+
+#[test]
+fn test_embed_file_7() {
+    let program = do_basic_run(&vec![
+        "run".to_string(),
+        "-i".to_string(),
+        "resources/tests".to_string(),
+        "(mod () (embed-file hello bin hello.bin) hello)".to_string(),
```

Comment:
Could make the name of the file different than the symbol name :)

Reply:
as in not hello.bin?